### ISSUE TITLE
Incorrect run test after tranning within google colab

### ISSUE THREAD

**Issue by TyrusRC** [cloud-annotations/object-detection-react#33]
Hi
I trained model which is about hand poses on google colab, but it seems not work correctlly. What should I do now?

**Comment by bourdakos1** [comment]
What is the error?

**Comment by TyrusRC** [comment]
This is the input image that I made on https://cloud.annotations.ai/.
![image](https://user-images.githubusercontent.com/63230297/104695715-bcfa2b80-573f-11eb-8d98-48b68ddf0fe5.png)
Moreover, I tried other poses but all predict result is letter K. 
![image](https://user-images.githubusercontent.com/63230297/104695461-6391fc80-573f-11eb-85be-19b0ee3072c2.png)

**Comment by bourdakos1** [comment]
Maybe you need more training data? You have a lot of classes that probably all look very similar, so you’ll need more training data for it to be able to distinguish

**Comment by TyrusRC** [comment]
Sure, I wonder how many images for each class that I need to add?

**Comment by bourdakos1** [comment]
I can’t say for sure, but try to get to 250 each and see if the accuracy improves

**Comment by TyrusRC** [comment]
Ok, thanks, I will try. In fact, I used same model for yolov5 and it runs correctly almost cases.

**Comment by bourdakos1** [comment]
Good to know, you could try playing around with the hyperparameters or other models to see if you can improve accuracy. The colab training process is new so there could be an issue on my end, I will look into it
